lenovo/thinkpad/general-intel.nix

```diff
@@ -0,0 +1,6 @@
+{ ... }:
+
+{
+  boot.kernelModules = mkDefault [ "kvm-intel" ];
+  services.xserver.videoDrivers = [ "intel" ];
```

Comment:
The ```intel``` driver is deprecated and all the distributions have moved to ```modesetting```.

Reply:
on x250 I still have freezes with `modsetting`

Reply:
Should we then ```mkDefault``` then for those who do want to use modesetting without having to ```mkForce```?

Reply:
I would only set that this for x250 like it was before with the comment.